packages/api/prisma/schema.prisma

```diff
@@ -15,13 +15,12 @@ datasource db {
 
 model Book {
   id     Int     @id
-  name   String
-  search String  @unique // This gives us a human readable key to look up books from URL params.
```

Comment:
Since the book names are going to be provided by the internationalization strings, we only need a string that we use for API calls and database lookups. I replaced `search` with `name` since that is a little clearer.